[refaktor] Update dependencies and use Vite

mlakatkou opened the pull request:
To ensure compatibility with new versions of dependencies and improve development experience, this commit migrates the project to use Vite as the build tool instead of the outdated Svelte template. Old Svelte template: https://github.com/sveltejs/template

mlakatkou committed:
[refaktor] Update dependencies and use Vite

To ensure compatibility with new versions of dependencies and improve development experience, this commit migrates the project to use Vite as the build tool instead of the deprecated Svelte template. Old Svelte template: https://github.com/sveltejs/template

mlakatkou committed:
[refaktor] Update dependencies and use Vite

To ensure compatibility with new versions of dependencies and improve development experience, this commit migrates the project to use Vite as the build tool instead of the deprecated Svelte template. Old Svelte template: https://github.com/sveltejs/template

mlakatkou committed:
Merge remote-tracking branch 'origin/main' into dev/1534

AlexKlimenkov commented on README.md:
```diff
@@ -6,19 +6,11 @@
 
 ## How to start
 
-Clone the repository or download files.
```
Probably unintended change, reverts a commit from main branch https://github.com/DHTMLX/svelte-event-calendar-demo/commit/91140ce08d9bdfeef2629a5fba798b7202209513
Rollback pls

AlexKlimenkov commented on index.html:
```diff
@@ -0,0 +1,13 @@
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + Svelte</title>
```
Pls add an appropriate title - `DHTMLX Event Calendar with Svelte` or something of sort

AlexKlimenkov reviewed (changes requested):
Revert changes in README.md and add a title to index.html

mlakatkou committed:
[update] Add npm instructions to the README file

To enhance project usability.

mlakatkou committed:
[update] Change title in the index.html

To improve the page's relevance.

mlakatkou committed:
[update] Change script in package.json

To improve development experience.

mlakatkou committed:
[update] Change instructions in the README file

To enhance project usability.

mlakatkou committed:
[update] Add destructor method

To remove all HTML elements of the Event Calendar and detach all related events after the Event Calendar component has been unmounted from the DOM.
